storage/innobase/handler/ha_innodb.cc

```diff
@@ -19727,7 +19736,7 @@ static MYSQL_SYSVAR_ULONG(page_hash_locks, srv_n_page_hash_locks,
 static MYSQL_SYSVAR_ULONG(doublewrite_batch_size, srv_doublewrite_batch_size,
   PLUGIN_VAR_OPCMDARG | PLUGIN_VAR_READONLY,
   "Number of pages reserved in doublewrite buffer for batch flushing",
-  NULL, NULL, 120, 1, 127, 0);
+  NULL, NULL, 120, 1, MAX_DOUBLEWRITE_BATCH_SIZE, 0);
```

Comment:
Cannot raise this above 127 without decoupling the legacy doublewrite buffer area size